src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+
 public class MainApp {
+    public static void main(String[] args) {
+        ArrayList<Machine> objectsOfMachine = new ArrayList<>();
+        objectsOfMachine.add(new Bulldozer());
+        objectsOfMachine.add(new Excavator());
+        objectsOfMachine.add(new Truck());
+        for (Machine el : objectsOfMachine) {
+            el.doWork();
+            el.stopWork();
+        }
```

Comment:
A few points here:
1) Pay attention to the task description - it requires you to put machines in the array, not an ArrayList
2) Naming: 
2.1) No need to be so specific in variable names, it's obvious that this variable will store objects of machine. In this case, we can be more general and say just "machines" or smth
2.2) In for you have the opposite situation. "el" name is too generic. If some other developer opens the 12th or 13th line he needs to go up to the 11th line to understand what el actually stores. So "machine" will be good enough in this case.